app/serializers/invite.js

```diff
@@ -1,10 +1,24 @@
+/* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
 import ApplicationSerializer from 'ghost-admin/serializers/application';
 import EmbeddedRecordsMixin from 'ember-data/serializers/embedded-records-mixin';
 
 export default ApplicationSerializer.extend(EmbeddedRecordsMixin, {
     attrs: {
-        roles: {embedded: 'always'},
+        role: {embedded: 'always'},
```

Comment:
This should be removed I think, the `{embedded: 'always'}` option is used when the API returns a full embedded record object whereas now we are only dealing with an ID so there's no special behaviour needed. The `EmbeddedRecordsMixin` can also be removed.